src/Struct/FindologicProduct.php

```diff
@@ -91,6 +92,9 @@ class FindologicProduct extends Struct
     /** @var Property[] */
     protected $properties;
 
+    /** @var Item */
+    protected $item;
```

Comment:
It would make sense to also change all the `private` setters like `setVariantPrices` to `protected`.